ui/pages/asset/asset.scss

```diff
@@ -31,3 +31,20 @@
     padding: 2px 0 2px 8px;
   }
 }
+
+.time-range-button {
+  height:24px;
+
+  &:hover {
+    background-color: transparent !important;
+    color: var(--color-text-alternative) !important;
+    box-shadow: var(--shadow-size-xs) var(--color-shadow-default) !important;
```

Comment:
Yes.  Went a little off script here to get the time range buttons styled like the designs.

Reply:
I was under the impression that since this stylesheet is loaded after the component-library stylesheet, the use of `!important` wouldn't be necessary, as the later stylesheet should naturally override the CSS rules due to specificity. @garrettbear, could you please investigate this further?

Reply:
In https://github.com/MetaMask/metamask-extension/pull/22508/commits/c28168ce4fbd0e9094504af6a2143521d4aa0978 I was able to remove all the `!important`s and simplify CSS by using `ButtonBase` instead of `ButtonSecondary`, which is what it was fighting.

I could simplify further if I knew how to access `--color-background-default-pressed` in JavaScipt, so that part remains CSS for now.

Reply:
![Screenshot_2024-03-04-10-18-02-37_40deb401b9ffe8e1df2f1cc5ba480b12.jpg](https://github.com/MetaMask/metamask-extension/assets/122599979/225d513a-3199-4c7b-91be-f68b5687723f)